contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,126 @@
+
+## More first-time founder mistakes
+
+### Not solving a real problem
+
+Exhibit A: [Humane AI Pin](https://www.theverge.com/24126502/humane-ai-pin-review)
+
+### The wrong co-founder(s)
+
+"[Companies are mostly about people](https://newsletter.posthog.com/p/what-we-learned-about-hiring-from)" and that starts with your co-founders. [Marty Kausus](https://www.linkedin.com/in/martykausas/), co-founder and CEO at [Pylon](https://usepylon.com), quit his second startup the moment he found "the right co-founders" to start a more successful venture.
+
+Marty and his co-founders got together and "pivoted weekly" for around two months before landing on Pylon, a Zendesk and Intercom alternative for managing customer operations via Slack. A year and a half later, they already have strong product-market fit.
+
+> "My biggest takeaway on co-founders is to make sure they have similar hustle, spirit, and desired lifestyle. For Pylon, we wanted to live together and live and breathe the company. We joke that we don't have "work-life balance", we have "work-life integration" and that's ideal for us, while most people would find that miserable."
+
+
+### Just shipping what users say they want
+
+Talking to customers is important, but it can go too far. This was a lesson [James Evans](https://www.linkedin.com/in/james-evans-7086b3126/), now co-founder at CEO at [CommandBar](https://www.commandbar.com), learned in his first startup:
+
+> "Our only real advantage relative to competitors was speed and eagerness, so we shipped a ton of features. Users would ask for something on Intercom in the morning and receive it that evening. Just crazy customer service."
+
+It felt good, but "created a ton of product debt" and led them to ship "things that we didn't really want users to use," James says. In the end, he argues, it made the product "kind of Franksteiny-freakish." 
+
+With CommandBar, James was determined to build a more opinionated product. They only build new features for customers when they can say, with confidence "this feature will help your users and not annoy them."
+
+### Poor founder-market fit
+
+[Per Borgen](https://www.linkedin.com/in/per-harald-borgen/) and his co-founder spent three years trying to make their first startup, Propell (2012 to 2015), work. Despite reaching over a million downloads for their children's books for iPad idea, the venture almost broke his spirit: "I was convinced I'd never have the energy to do a startup again."
+
+In his mind, they made two critical errors:
+
+1. The market was too small.
+2. They didn't have a "strong passion" for the problem they were trying to solve.
+
+They rectified both with [Scrimba](https://scrimba.com/), an online coding school they founded in 2016. It's profitable with a team of 15 and has over 10,000 customers. 
```

Comment:
How did they rectify this?

Reply:
Answer is in the next line, but can make this more explicit.